tools/llvm-spirv/llvm-spirv.cpp

```diff
@@ -202,6 +202,12 @@ static cl::opt<SPIRV::DebugInfoEIS> DebugEIS(
                    "extended instruction set. This version of SPIR-V debug "
                    "info format is compatible with the SPIRV-Tools")));
 
+static cl::opt<bool> SPIRVReplaceLLVMFmulAddWithOpenCLMad(
```

Comment:
Is there a reason to make it a `bool`-accepting option instead of a simple flag?
(I personally don't think it matters much, but just for the sake of consistency)

Reply:
> Is there a reason to make it a `bool`-accepting option instead of a simple flag?

No particular reason from command line interface point of view, but from technical point of view I need to take a more careful look at LLVM's CommandLine library to see if it is possible

Reply:
Looks like [`boolOrDefault`](https://llvm.org/docs/CommandLine.html#boolean-flags) is what I'm looking for here

Reply:
Ok, I was too quick with `boolOrDefault`: I don't need it, because I don't really care about whether the option was passed or not, but I care about its value.

Also, CommandLine library [supports](https://llvm.org/docs/CommandLine.html#boolean-arguments) flags through boolean options, i.e. `-spirv-replace-fmuladd-with-ocl-mad` is equivalent to `-spirv-replace-fmuladd-with-ocl-mad=true`.

From my point of view the current way is good enough: most likely no one will ever use that switch and I don't want to overcomplicate the code just to get a `-no-spirv-replace-fmuladd-with-ocl-mad` alias (moreover, due to CommandLine library it would be really confusing to be able to use `-no-spirv-replace-fmuladd-with-ocl-mad=false`)

Reply:
Thanks for expanding on this, SGTM.